R/tibble.R

```diff
@@ -143,6 +143,28 @@ tibble <- function(...,
   tibble_quos(xs[!is_null], .rows, .name_repair)
 }
 
+#' tibble_row()
+#'
+#' @description
+#' `tibble_row()` constructs a data frame that is guaranteed to occupy one row.
+#' Vector columns are required to have size one, non-vector columns are wrapped
+#' in a list.
+#'
+#' @rdname tibble
+#' @export
+#' @examples
+#'
+#' # Use tibble_row() to construct a one-row tibble:
+#' tibble_row(a = 1, lm = lm(Petal.Width ~ Petal.Length + Species, data = iris))
+tibble_row <- function(...,
+                       .name_repair = c("check_unique", "unique", "universal", "minimal")) {
+  xs <- quos(...)
```

Comment:
Is it clear that we want the special evaluation here?

Reply:
It looks useful to me. For standard evaluation we can have `as_tibble_row()` with similar semantics.

What do you think?

Reply:
Ok, makes sense